lms/envs/common.py

```diff
@@ -233,10 +233,10 @@
     'ENABLED_PAYMENT_REPORTS': ["refund_report", "itemized_purchase_report", "university_revenue_share", "certificate_status"],
 
     # Turn off account locking if failed login attempts exceeds a limit
-    'ENABLE_MAX_FAILED_LOGIN_ATTEMPTS': False,
+    'ENABLE_MAX_FAILED_LOGIN_ATTEMPTS': True,
```

Comment:
There's a minimum wait period which is a CONFIG setting. I think the default is 15 minutes.